sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/LiteralExpressionSuite.scala

```diff
@@ -279,4 +280,40 @@ class LiteralExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkEvaluation(Literal(Array(instant0, instant1)), Array(instant0, instant1))
     }
   }
+
+  private def withTimeZones(
+      sessionTimeZone: String,
+      systemTimeZone: String)(f: => Unit): Unit = {
```

Comment:
Why passing in `systemTimeZone`? It isn't used in `withTimeZones`.

Reply:
I forgot to substitute value in `TimeZone.setDefault` when extracted the common code. @viirya Thank you for pointing this out.